float64 cur_lat			# Info of current and target latitudes and longitudes
float64 tar_lat
float64 cur_long
float64 tar_long
float64 heading			# Current orientation of the robot
